src/hotspot/share/gc/shared/collectedHeap.hpp

```diff
@@ -215,6 +215,11 @@ class CollectedHeap : public CHeapObj<mtInternal> {
 
   virtual size_t capacity() const = 0;
   virtual size_t used() const = 0;
+  // Returns the estimate of live set size. Because live set changes over time,
```

Comment:
I believe a blank line is in order here, look at other comments in the same header.

Reply:
Done